agent/lib/kontena/workers/node_info_worker.rb

```diff
@@ -49,21 +49,22 @@ def on_node_info(topic, node)
 
     def publish_node_info
       debug 'publishing node information'
-      node_info = docker_info.dup
+      node_info = Docker.info()
```

Comment:
I vaugely like the parens for things that are actual method calls with side-effects, not just cheap attribute accesses... TBH I also think those should be verbs, like `Docker.get_info()`